core/src/main/java/org/mskcc/cbio/portal/servlet/CancerStudyView.java

```diff
@@ -140,16 +138,12 @@ protected void processRequest(HttpServletRequest request, HttpServletResponse re
      * @param cancerStudyId
      * @return cancerStudy
      */
-    private CancerStudy getCancerStudyDetails(String cancerStudyId) throws NotAuthorizedException, DaoException {
```

Comment:
this method now doesn't throw any error since study accessibility isn't done here. Its handled at https://github.com/cBioPortal/cbioportal/pull/3631/files/0f02e898d60ac62ca986406a1d265f0795f9a976#diff-c17cf799714af30e11d56ccb957974f9R187